content/minutes/cab/2022-08-17-minutes.md

```diff
@@ -0,0 +1,34 @@
+---
+date: "2022-08-17T00:00:00+00:00"
+title: "August 17, 2022 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Diego Nolasco"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/about/organization/#open-force-field-consortium) met on August 17, 2022.
+The minutes are summarized as follows:
+
+#### Rosemary + The Tresadern paper review
+
+* D. Mobley announced that the Rosemary + Tresadern manuscript is in the final stages of revision and asked industrial partners who had not yet made their comments to do so. The deadline is September 17th.
+
+#### Boehringer Ingelheim and Merck are discontinuing their support to OpenFF
+
+* D. Nolasco announced that Boehringer Ingelheim and Merck are discontinuing their support to OpenFF, but will continue supporting OpenFE.
+
+#### New version of Sage (openff-2.1.0)
+
+* D. Nolasco held a consultation with board members to see if there is any interest in the release a new version of Sage (openff-2.1.0). The present members pointed out that a new version would not add value at the moment, but suggested that we seek to understand what are the gaps in science about force fields that we could attack in the interest of continuing to add value to the molecular simularion community.
+
+#### Follow up workshops
+
+* J. Wagner announced that the follow up workshops will be scheduled for September and October. We will probably held three workshops: BespokeFit, Interchange and Interactive force field hacking - protein parameterization with PTMs.
+
+#### New version of the annual report
+
+* D. Mobley announced the availability of a new version of the annual report. The document addresses some shortcomings pointed out by Katharina Meier and Thomas Fox at the last governing board meeting.
```

Comment:
Are we posting a public copy of the annual report anywhere? Could be good to link it in this line if so.

Reply:
I'll link the Zenodo link to the old version, and update it as soon as we have the new version available.